[2021-12-29 11:57:35,022] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: scraping_daft_mayo.scrap_data scheduled__2021-12-28T10:20:00+00:00 [queued]>
[2021-12-29 11:57:36,235] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: scraping_daft_mayo.scrap_data scheduled__2021-12-28T10:20:00+00:00 [queued]>
[2021-12-29 11:57:36,269] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-12-29 11:57:36,301] {taskinstance.py:1242} INFO - Starting attempt 1 of 4
[2021-12-29 11:57:36,385] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-12-29 11:57:36,583] {taskinstance.py:1262} INFO - Executing <Task(SparkSubmitOperator): scrap_data> on 2021-12-28 10:20:00+00:00
[2021-12-29 11:57:36,591] {standard_task_runner.py:52} INFO - Started process 24742 to run task
[2021-12-29 11:57:36,696] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'scraping_daft_mayo', 'scrap_data', 'scheduled__2021-12-28T10:20:00+00:00', '--job-id', '5333', '--raw', '--subdir', 'DAGS_FOLDER/scraping_daft_mayo.py', '--cfg-path', '/tmp/tmphmuno5nk', '--error-file', '/tmp/tmpko8ismbo']
[2021-12-29 11:57:36,707] {standard_task_runner.py:77} INFO - Job 5333: Subtask scrap_data
[2021-12-29 11:57:37,843] {logging_mixin.py:109} INFO - Running <TaskInstance: scraping_daft_mayo.scrap_data scheduled__2021-12-28T10:20:00+00:00 [running]> on host 6bdda4c19a85
[2021-12-29 11:57:39,897] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=clement.liscoet@gmail.com
AIRFLOW_CTX_DAG_OWNER=clemoni
AIRFLOW_CTX_DAG_ID=scraping_daft_mayo
AIRFLOW_CTX_TASK_ID=scrap_data
AIRFLOW_CTX_EXECUTION_DATE=2021-12-28T10:20:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-28T10:20:00+00:00
[2021-12-29 11:57:40,016] {base.py:79} INFO - Using connection to: id: spark_default. Host: spark://spark:7077, Port: None, Schema: , Login: ***, Password: ***, extra: {}
[2021-12-29 11:57:40,025] {spark_submit.py:360} INFO - Spark-Submit cmd: spark-submit --master spark://spark:7077 --conf spark.master=spark://spark:7077 --name scrap data --verbose /opt/***/jobs/get_ads_per_county.py mayo
[2021-12-29 11:59:53,631] {spark_submit.py:514} INFO - Using properties file: null
[2021-12-29 11:59:57,800] {spark_submit.py:514} INFO - WARNING: An illegal reflective access operation has occurred
[2021-12-29 11:59:57,802] {spark_submit.py:514} INFO - WARNING: Illegal reflective access by org.apache.spark.unsafe.Platform (file:/home/***/.local/lib/python3.6/site-packages/pyspark/jars/spark-unsafe_2.12-3.2.0.jar) to constructor java.nio.DirectByteBuffer(long,int)
[2021-12-29 11:59:57,804] {spark_submit.py:514} INFO - WARNING: Please consider reporting this to the maintainers of org.apache.spark.unsafe.Platform
[2021-12-29 11:59:57,806] {spark_submit.py:514} INFO - WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
[2021-12-29 11:59:57,808] {spark_submit.py:514} INFO - WARNING: All illegal access operations will be denied in a future release
[2021-12-29 11:59:58,229] {spark_submit.py:514} INFO - Parsed arguments:
[2021-12-29 11:59:58,231] {spark_submit.py:514} INFO - master                  spark://spark:7077
[2021-12-29 11:59:58,236] {spark_submit.py:514} INFO - deployMode              null
[2021-12-29 11:59:58,240] {spark_submit.py:514} INFO - executorMemory          null
[2021-12-29 11:59:58,242] {spark_submit.py:514} INFO - executorCores           null
[2021-12-29 11:59:58,250] {spark_submit.py:514} INFO - totalExecutorCores      null
[2021-12-29 11:59:58,252] {spark_submit.py:514} INFO - propertiesFile          null
[2021-12-29 11:59:58,259] {spark_submit.py:514} INFO - driverMemory            null
[2021-12-29 11:59:58,264] {spark_submit.py:514} INFO - driverCores             null
[2021-12-29 11:59:58,270] {spark_submit.py:514} INFO - driverExtraClassPath    null
[2021-12-29 11:59:58,278] {spark_submit.py:514} INFO - driverExtraLibraryPath  null
[2021-12-29 11:59:58,284] {spark_submit.py:514} INFO - driverExtraJavaOptions  null
[2021-12-29 11:59:58,290] {spark_submit.py:514} INFO - supervise               false
[2021-12-29 11:59:58,296] {spark_submit.py:514} INFO - queue                   null
[2021-12-29 11:59:58,303] {spark_submit.py:514} INFO - numExecutors            null
[2021-12-29 11:59:58,309] {spark_submit.py:514} INFO - files                   null
[2021-12-29 11:59:58,316] {spark_submit.py:514} INFO - pyFiles                 null
[2021-12-29 11:59:58,322] {spark_submit.py:514} INFO - archives                null
[2021-12-29 11:59:58,329] {spark_submit.py:514} INFO - mainClass               null
[2021-12-29 11:59:58,333] {spark_submit.py:514} INFO - primaryResource         file:/opt/***/jobs/get_ads_per_county.py
[2021-12-29 11:59:58,341] {spark_submit.py:514} INFO - name                    scrap data
[2021-12-29 11:59:58,348] {spark_submit.py:514} INFO - childArgs               [mayo]
[2021-12-29 11:59:58,352] {spark_submit.py:514} INFO - jars                    null
[2021-12-29 11:59:58,361] {spark_submit.py:514} INFO - packages                null
[2021-12-29 11:59:58,369] {spark_submit.py:514} INFO - packagesExclusions      null
[2021-12-29 11:59:58,373] {spark_submit.py:514} INFO - repositories            null
[2021-12-29 11:59:58,379] {spark_submit.py:514} INFO - verbose                 true
[2021-12-29 11:59:58,385] {spark_submit.py:514} INFO - 
[2021-12-29 11:59:58,389] {spark_submit.py:514} INFO - Spark properties used, including those specified through
[2021-12-29 11:59:58,391] {spark_submit.py:514} INFO - --conf and those from the properties file null:
[2021-12-29 11:59:58,394] {spark_submit.py:514} INFO - (spark.master,spark://spark:7077)
[2021-12-29 11:59:58,397] {spark_submit.py:514} INFO - 
[2021-12-29 11:59:58,399] {spark_submit.py:514} INFO - 
[2021-12-29 12:00:04,385] {spark_submit.py:514} INFO - Main class:
[2021-12-29 12:00:04,393] {spark_submit.py:514} INFO - org.apache.spark.deploy.PythonRunner
[2021-12-29 12:00:04,398] {spark_submit.py:514} INFO - Arguments:
[2021-12-29 12:00:04,400] {spark_submit.py:514} INFO - file:/opt/***/jobs/get_ads_per_county.py
[2021-12-29 12:00:04,405] {spark_submit.py:514} INFO - null
[2021-12-29 12:00:04,414] {spark_submit.py:514} INFO - mayo
[2021-12-29 12:00:04,417] {spark_submit.py:514} INFO - --verbose
[2021-12-29 12:00:04,429] {spark_submit.py:514} INFO - Spark config:
[2021-12-29 12:00:04,432] {spark_submit.py:514} INFO - (spark.app.name,scrap data)
[2021-12-29 12:00:04,435] {spark_submit.py:514} INFO - (spark.master,spark://spark:7077)
[2021-12-29 12:00:04,439] {spark_submit.py:514} INFO - (spark.submit.pyFiles,)
[2021-12-29 12:00:04,442] {spark_submit.py:514} INFO - (spark.submit.deployMode,client)
[2021-12-29 12:00:04,448] {spark_submit.py:514} INFO - Classpath elements:
[2021-12-29 12:00:04,449] {spark_submit.py:514} INFO - 
[2021-12-29 12:00:04,457] {spark_submit.py:514} INFO - 
[2021-12-29 12:00:04,459] {spark_submit.py:514} INFO - 
[2021-12-29 12:00:18,815] {spark_submit.py:514} INFO - Using Spark's default log4j profile: org/apache/spark/log4j-defaults.properties
[2021-12-29 12:00:19,007] {spark_submit.py:514} INFO - 21/12/29 12:00:18 INFO SparkContext: Running Spark version 3.2.0
[2021-12-29 12:00:22,117] {spark_submit.py:514} INFO - 21/12/29 12:00:22 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2021-12-29 12:00:27,474] {spark_submit.py:514} INFO - 21/12/29 12:00:27 INFO ResourceUtils: ==============================================================
[2021-12-29 12:00:27,487] {spark_submit.py:514} INFO - 21/12/29 12:00:27 INFO ResourceUtils: No custom resources configured for spark.driver.
[2021-12-29 12:00:27,502] {spark_submit.py:514} INFO - 21/12/29 12:00:27 INFO ResourceUtils: ==============================================================
[2021-12-29 12:00:27,539] {spark_submit.py:514} INFO - 21/12/29 12:00:27 INFO SparkContext: Submitted application: Get Ads per county
[2021-12-29 12:00:28,360] {spark_submit.py:514} INFO - 21/12/29 12:00:28 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(cores -> name: cores, amount: 1, script: , vendor: , memory -> name: memory, amount: 1024, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2021-12-29 12:00:28,425] {spark_submit.py:514} INFO - 21/12/29 12:00:28 INFO ResourceProfile: Limiting resource is cpu
[2021-12-29 12:00:28,427] {spark_submit.py:514} INFO - 21/12/29 12:00:28 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2021-12-29 12:00:29,955] {spark_submit.py:514} INFO - 21/12/29 12:00:29 INFO SecurityManager: Changing view acls to: ***
[2021-12-29 12:00:29,961] {spark_submit.py:514} INFO - 21/12/29 12:00:29 INFO SecurityManager: Changing modify acls to: ***
[2021-12-29 12:00:29,962] {spark_submit.py:514} INFO - 21/12/29 12:00:29 INFO SecurityManager: Changing view acls groups to:
[2021-12-29 12:00:29,966] {spark_submit.py:514} INFO - 21/12/29 12:00:29 INFO SecurityManager: Changing modify acls groups to:
[2021-12-29 12:00:29,969] {spark_submit.py:514} INFO - 21/12/29 12:00:29 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users  with view permissions: Set(***); groups with view permissions: Set(); users  with modify permissions: Set(***); groups with modify permissions: Set()
[2021-12-29 12:00:38,017] {spark_submit.py:514} INFO - 21/12/29 12:00:38 INFO Utils: Successfully started service 'sparkDriver' on port 44849.
[2021-12-29 12:00:38,541] {spark_submit.py:514} INFO - 21/12/29 12:00:38 INFO SparkEnv: Registering MapOutputTracker
[2021-12-29 12:00:41,305] {spark_submit.py:514} INFO - 21/12/29 12:00:41 INFO SparkEnv: Registering BlockManagerMaster
[2021-12-29 12:00:42,165] {spark_submit.py:514} INFO - 21/12/29 12:00:42 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2021-12-29 12:00:42,175] {spark_submit.py:514} INFO - 21/12/29 12:00:42 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2021-12-29 12:00:42,411] {spark_submit.py:514} INFO - 21/12/29 12:00:42 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2021-12-29 12:00:44,287] {spark_submit.py:514} INFO - 21/12/29 12:00:44 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-343840ab-aaf8-4d25-830d-ef9bf751979e
[2021-12-29 12:00:44,608] {spark_submit.py:514} INFO - 21/12/29 12:00:44 INFO MemoryStore: MemoryStore started with capacity 434.4 MiB
[2021-12-29 12:00:45,283] {spark_submit.py:514} INFO - 21/12/29 12:00:45 INFO SparkEnv: Registering OutputCommitCoordinator
[2021-12-29 12:01:04,953] {spark_submit.py:514} INFO - 21/12/29 12:01:04 WARN Utils: Service 'SparkUI' could not bind on port 4040. Attempting port 4041.
[2021-12-29 12:01:04,984] {spark_submit.py:514} INFO - 21/12/29 12:01:04 WARN Utils: Service 'SparkUI' could not bind on port 4041. Attempting port 4042.
[2021-12-29 12:01:05,010] {spark_submit.py:514} INFO - 21/12/29 12:01:04 WARN Utils: Service 'SparkUI' could not bind on port 4042. Attempting port 4043.
[2021-12-29 12:01:05,055] {spark_submit.py:514} INFO - 21/12/29 12:01:05 WARN Utils: Service 'SparkUI' could not bind on port 4043. Attempting port 4044.
[2021-12-29 12:01:05,123] {spark_submit.py:514} INFO - 21/12/29 12:01:05 WARN Utils: Service 'SparkUI' could not bind on port 4044. Attempting port 4045.
[2021-12-29 12:01:05,349] {spark_submit.py:514} INFO - 21/12/29 12:01:05 WARN Utils: Service 'SparkUI' could not bind on port 4045. Attempting port 4046.
[2021-12-29 12:01:05,446] {spark_submit.py:514} INFO - 21/12/29 12:01:05 WARN Utils: Service 'SparkUI' could not bind on port 4046. Attempting port 4047.
[2021-12-29 12:01:05,782] {spark_submit.py:514} INFO - 21/12/29 12:01:05 WARN Utils: Service 'SparkUI' could not bind on port 4047. Attempting port 4048.
[2021-12-29 12:01:05,898] {spark_submit.py:514} INFO - 21/12/29 12:01:05 WARN Utils: Service 'SparkUI' could not bind on port 4048. Attempting port 4049.
[2021-12-29 12:01:06,133] {spark_submit.py:514} INFO - 21/12/29 12:01:05 WARN Utils: Service 'SparkUI' could not bind on port 4049. Attempting port 4050.
[2021-12-29 12:01:06,302] {spark_submit.py:514} INFO - 21/12/29 12:01:05 WARN Utils: Service 'SparkUI' could not bind on port 4050. Attempting port 4051.
[2021-12-29 12:01:06,316] {spark_submit.py:514} INFO - 21/12/29 12:01:05 WARN Utils: Service 'SparkUI' could not bind on port 4051. Attempting port 4052.
[2021-12-29 12:01:06,342] {spark_submit.py:514} INFO - 21/12/29 12:01:05 WARN Utils: Service 'SparkUI' could not bind on port 4052. Attempting port 4053.
[2021-12-29 12:01:09,875] {spark_submit.py:514} INFO - 21/12/29 12:01:09 INFO Utils: Successfully started service 'SparkUI' on port 4053.
[2021-12-29 12:01:13,857] {spark_submit.py:514} INFO - 21/12/29 12:01:13 INFO SparkUI: Bound SparkUI to 0.0.0.0, and started at http://6bdda4c19a85:4053
[2021-12-29 12:02:15,219] {spark_submit.py:514} INFO - 21/12/29 12:02:15 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark:7077...
[2021-12-29 12:02:20,450] {spark_submit.py:514} INFO - 21/12/29 12:02:20 WARN StandaloneAppClient$ClientEndpoint: Failed to connect to master spark:7077
[2021-12-29 12:02:22,718] {spark_submit.py:514} INFO - org.apache.spark.SparkException: Exception thrown in awaitResult:
[2021-12-29 12:02:26,919] {spark_submit.py:514} INFO - at org.apache.spark.util.ThreadUtils$.awaitResult(ThreadUtils.scala:301)
[2021-12-29 12:02:26,963] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcTimeout.awaitResult(RpcTimeout.scala:75)
[2021-12-29 12:02:26,968] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRefByURI(RpcEnv.scala:101)
[2021-12-29 12:02:26,978] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRef(RpcEnv.scala:109)
[2021-12-29 12:02:28,009] {spark_submit.py:514} INFO - at org.apache.spark.deploy.client.StandaloneAppClient$ClientEndpoint$$anon$1.run(StandaloneAppClient.scala:107)
[2021-12-29 12:02:28,119] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[2021-12-29 12:02:28,179] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[2021-12-29 12:02:28,264] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2021-12-29 12:02:29,063] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2021-12-29 12:02:29,080] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-29 12:02:29,264] {spark_submit.py:514} INFO - Caused by: java.io.IOException: Failed to connect to spark:7077
[2021-12-29 12:02:29,272] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:288)
[2021-12-29 12:02:29,275] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:218)
[2021-12-29 12:02:29,281] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:230)
[2021-12-29 12:02:29,290] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.NettyRpcEnv.createClient(NettyRpcEnv.scala:204)
[2021-12-29 12:02:29,297] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:202)
[2021-12-29 12:02:29,301] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:198)
[2021-12-29 12:02:29,304] {spark_submit.py:514} INFO - ... 4 more
[2021-12-29 12:02:29,311] {spark_submit.py:514} INFO - Caused by: java.net.UnknownHostException: spark
[2021-12-29 12:02:29,314] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress$CachedAddresses.get(InetAddress.java:797)
[2021-12-29 12:02:29,320] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1509)
[2021-12-29 12:02:29,323] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1368)
[2021-12-29 12:02:29,391] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1302)
[2021-12-29 12:02:29,405] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getByName(InetAddress.java:1252)
[2021-12-29 12:02:29,413] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:156)
[2021-12-29 12:02:29,419] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:153)
[2021-12-29 12:02:29,448] {spark_submit.py:514} INFO - at java.base/java.security.AccessController.doPrivileged(Native Method)
[2021-12-29 12:02:29,453] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils.addressByName(SocketUtils.java:153)
[2021-12-29 12:02:29,456] {spark_submit.py:514} INFO - at io.netty.resolver.DefaultNameResolver.doResolve(DefaultNameResolver.java:41)
[2021-12-29 12:02:29,458] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:61)
[2021-12-29 12:02:29,460] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:53)
[2021-12-29 12:02:29,464] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:55)
[2021-12-29 12:02:29,465] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:31)
[2021-12-29 12:02:29,467] {spark_submit.py:514} INFO - at io.netty.resolver.AbstractAddressResolver.resolve(AbstractAddressResolver.java:106)
[2021-12-29 12:02:29,481] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.doResolveAndConnect0(Bootstrap.java:206)
[2021-12-29 12:02:29,501] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.access$000(Bootstrap.java:46)
[2021-12-29 12:02:29,504] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:180)
[2021-12-29 12:02:29,511] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:166)
[2021-12-29 12:02:29,520] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:578)
[2021-12-29 12:02:29,527] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:552)
[2021-12-29 12:02:29,543] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:491)
[2021-12-29 12:02:29,547] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setValue0(DefaultPromise.java:616)
[2021-12-29 12:02:29,555] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setSuccess0(DefaultPromise.java:605)
[2021-12-29 12:02:29,570] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.trySuccess(DefaultPromise.java:104)
[2021-12-29 12:02:29,576] {spark_submit.py:514} INFO - at io.netty.channel.DefaultChannelPromise.trySuccess(DefaultChannelPromise.java:84)
[2021-12-29 12:02:29,580] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.safeSetSuccess(AbstractChannel.java:1008)
[2021-12-29 12:02:29,583] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:516)
[2021-12-29 12:02:29,586] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.access$200(AbstractChannel.java:429)
[2021-12-29 12:02:29,588] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:486)
[2021-12-29 12:02:29,590] {spark_submit.py:514} INFO - at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164)
[2021-12-29 12:02:29,593] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:469)
[2021-12-29 12:02:29,595] {spark_submit.py:514} INFO - at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:500)
[2021-12-29 12:02:29,598] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
[2021-12-29 12:02:29,606] {spark_submit.py:514} INFO - at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[2021-12-29 12:02:29,615] {spark_submit.py:514} INFO - at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
[2021-12-29 12:02:29,618] {spark_submit.py:514} INFO - ... 1 more
[2021-12-29 12:02:34,755] {spark_submit.py:514} INFO - 21/12/29 12:02:34 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark:7077...
[2021-12-29 12:02:37,360] {spark_submit.py:514} INFO - 21/12/29 12:02:37 WARN StandaloneAppClient$ClientEndpoint: Failed to connect to master spark:7077
[2021-12-29 12:02:37,378] {spark_submit.py:514} INFO - org.apache.spark.SparkException: Exception thrown in awaitResult:
[2021-12-29 12:02:37,384] {spark_submit.py:514} INFO - at org.apache.spark.util.ThreadUtils$.awaitResult(ThreadUtils.scala:301)
[2021-12-29 12:02:37,392] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcTimeout.awaitResult(RpcTimeout.scala:75)
[2021-12-29 12:02:37,396] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRefByURI(RpcEnv.scala:101)
[2021-12-29 12:02:37,399] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRef(RpcEnv.scala:109)
[2021-12-29 12:02:37,402] {spark_submit.py:514} INFO - at org.apache.spark.deploy.client.StandaloneAppClient$ClientEndpoint$$anon$1.run(StandaloneAppClient.scala:107)
[2021-12-29 12:02:37,407] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[2021-12-29 12:02:37,408] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[2021-12-29 12:02:37,410] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2021-12-29 12:02:37,416] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2021-12-29 12:02:37,420] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-29 12:02:37,427] {spark_submit.py:514} INFO - Caused by: java.io.IOException: Failed to connect to spark:7077
[2021-12-29 12:02:37,429] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:288)
[2021-12-29 12:02:37,432] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:218)
[2021-12-29 12:02:37,439] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:230)
[2021-12-29 12:02:37,460] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.NettyRpcEnv.createClient(NettyRpcEnv.scala:204)
[2021-12-29 12:02:37,479] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:202)
[2021-12-29 12:02:37,484] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:198)
[2021-12-29 12:02:37,486] {spark_submit.py:514} INFO - ... 4 more
[2021-12-29 12:02:37,503] {spark_submit.py:514} INFO - Caused by: java.net.UnknownHostException: spark
[2021-12-29 12:02:37,509] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress$CachedAddresses.get(InetAddress.java:797)
[2021-12-29 12:02:37,513] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1509)
[2021-12-29 12:02:37,527] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1368)
[2021-12-29 12:02:37,548] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1302)
[2021-12-29 12:02:37,554] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getByName(InetAddress.java:1252)
[2021-12-29 12:02:37,557] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:156)
[2021-12-29 12:02:37,561] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:153)
[2021-12-29 12:02:37,568] {spark_submit.py:514} INFO - at java.base/java.security.AccessController.doPrivileged(Native Method)
[2021-12-29 12:02:37,582] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils.addressByName(SocketUtils.java:153)
[2021-12-29 12:02:37,606] {spark_submit.py:514} INFO - at io.netty.resolver.DefaultNameResolver.doResolve(DefaultNameResolver.java:41)
[2021-12-29 12:02:37,618] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:61)
[2021-12-29 12:02:37,622] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:53)
[2021-12-29 12:02:37,627] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:55)
[2021-12-29 12:02:37,637] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:31)
[2021-12-29 12:02:37,640] {spark_submit.py:514} INFO - at io.netty.resolver.AbstractAddressResolver.resolve(AbstractAddressResolver.java:106)
[2021-12-29 12:02:37,647] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.doResolveAndConnect0(Bootstrap.java:206)
[2021-12-29 12:02:37,656] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.access$000(Bootstrap.java:46)
[2021-12-29 12:02:37,770] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:180)
[2021-12-29 12:02:37,779] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:166)
[2021-12-29 12:02:37,789] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:578)
[2021-12-29 12:02:37,794] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:552)
[2021-12-29 12:02:37,795] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:491)
[2021-12-29 12:02:37,797] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setValue0(DefaultPromise.java:616)
[2021-12-29 12:02:37,802] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setSuccess0(DefaultPromise.java:605)
[2021-12-29 12:02:37,814] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.trySuccess(DefaultPromise.java:104)
[2021-12-29 12:02:37,818] {spark_submit.py:514} INFO - at io.netty.channel.DefaultChannelPromise.trySuccess(DefaultChannelPromise.java:84)
[2021-12-29 12:02:37,826] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.safeSetSuccess(AbstractChannel.java:1008)
[2021-12-29 12:02:37,837] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:516)
[2021-12-29 12:02:37,845] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.access$200(AbstractChannel.java:429)
[2021-12-29 12:02:37,847] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:486)
[2021-12-29 12:02:37,848] {spark_submit.py:514} INFO - at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164)
[2021-12-29 12:02:37,855] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:469)
[2021-12-29 12:02:37,865] {spark_submit.py:514} INFO - at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:500)
[2021-12-29 12:02:37,869] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
[2021-12-29 12:02:37,872] {spark_submit.py:514} INFO - at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[2021-12-29 12:02:37,874] {spark_submit.py:514} INFO - at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
[2021-12-29 12:02:37,882] {spark_submit.py:514} INFO - ... 1 more
[2021-12-29 12:02:54,393] {spark_submit.py:514} INFO - 21/12/29 12:02:54 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark:7077...
[2021-12-29 12:02:54,698] {spark_submit.py:514} INFO - 21/12/29 12:02:54 WARN StandaloneAppClient$ClientEndpoint: Failed to connect to master spark:7077
[2021-12-29 12:02:54,706] {spark_submit.py:514} INFO - org.apache.spark.SparkException: Exception thrown in awaitResult:
[2021-12-29 12:02:54,713] {spark_submit.py:514} INFO - at org.apache.spark.util.ThreadUtils$.awaitResult(ThreadUtils.scala:301)
[2021-12-29 12:02:54,716] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcTimeout.awaitResult(RpcTimeout.scala:75)
[2021-12-29 12:02:54,720] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRefByURI(RpcEnv.scala:101)
[2021-12-29 12:02:54,722] {spark_submit.py:514} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRef(RpcEnv.scala:109)
[2021-12-29 12:02:54,725] {spark_submit.py:514} INFO - at org.apache.spark.deploy.client.StandaloneAppClient$ClientEndpoint$$anon$1.run(StandaloneAppClient.scala:107)
[2021-12-29 12:02:54,727] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[2021-12-29 12:02:54,730] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[2021-12-29 12:02:54,732] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2021-12-29 12:02:54,734] {spark_submit.py:514} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2021-12-29 12:02:54,737] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-29 12:02:54,740] {spark_submit.py:514} INFO - Caused by: java.io.IOException: Failed to connect to spark:7077
[2021-12-29 12:02:54,742] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:288)
[2021-12-29 12:02:54,745] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:218)
[2021-12-29 12:02:54,748] {spark_submit.py:514} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:230)
[2021-12-29 12:02:54,753] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.NettyRpcEnv.createClient(NettyRpcEnv.scala:204)
[2021-12-29 12:02:54,756] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:202)
[2021-12-29 12:02:54,759] {spark_submit.py:514} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:198)
[2021-12-29 12:02:54,761] {spark_submit.py:514} INFO - ... 4 more
[2021-12-29 12:02:54,763] {spark_submit.py:514} INFO - Caused by: java.net.UnknownHostException: spark
[2021-12-29 12:02:54,773] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress$CachedAddresses.get(InetAddress.java:797)
[2021-12-29 12:02:54,775] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1509)
[2021-12-29 12:02:54,777] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1368)
[2021-12-29 12:02:54,779] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1302)
[2021-12-29 12:02:54,781] {spark_submit.py:514} INFO - at java.base/java.net.InetAddress.getByName(InetAddress.java:1252)
[2021-12-29 12:02:54,783] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:156)
[2021-12-29 12:02:54,792] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:153)
[2021-12-29 12:02:54,795] {spark_submit.py:514} INFO - at java.base/java.security.AccessController.doPrivileged(Native Method)
[2021-12-29 12:02:54,813] {spark_submit.py:514} INFO - at io.netty.util.internal.SocketUtils.addressByName(SocketUtils.java:153)
[2021-12-29 12:02:54,823] {spark_submit.py:514} INFO - at io.netty.resolver.DefaultNameResolver.doResolve(DefaultNameResolver.java:41)
[2021-12-29 12:02:54,826] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:61)
[2021-12-29 12:02:54,832] {spark_submit.py:514} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:53)
[2021-12-29 12:02:54,837] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:55)
[2021-12-29 12:02:54,839] {spark_submit.py:514} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:31)
[2021-12-29 12:02:54,841] {spark_submit.py:514} INFO - at io.netty.resolver.AbstractAddressResolver.resolve(AbstractAddressResolver.java:106)
[2021-12-29 12:02:54,842] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.doResolveAndConnect0(Bootstrap.java:206)
[2021-12-29 12:02:54,844] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap.access$000(Bootstrap.java:46)
[2021-12-29 12:02:54,846] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:180)
[2021-12-29 12:02:54,849] {spark_submit.py:514} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:166)
[2021-12-29 12:02:54,851] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:578)
[2021-12-29 12:02:54,854] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:552)
[2021-12-29 12:02:54,856] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:491)
[2021-12-29 12:02:54,859] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setValue0(DefaultPromise.java:616)
[2021-12-29 12:02:54,866] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.setSuccess0(DefaultPromise.java:605)
[2021-12-29 12:02:54,868] {spark_submit.py:514} INFO - at io.netty.util.concurrent.DefaultPromise.trySuccess(DefaultPromise.java:104)
[2021-12-29 12:02:54,870] {spark_submit.py:514} INFO - at io.netty.channel.DefaultChannelPromise.trySuccess(DefaultChannelPromise.java:84)
[2021-12-29 12:02:54,872] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.safeSetSuccess(AbstractChannel.java:1008)
[2021-12-29 12:02:54,873] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:516)
[2021-12-29 12:02:54,875] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.access$200(AbstractChannel.java:429)
[2021-12-29 12:02:54,877] {spark_submit.py:514} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:486)
[2021-12-29 12:02:54,879] {spark_submit.py:514} INFO - at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164)
[2021-12-29 12:02:54,880] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:469)
[2021-12-29 12:02:54,882] {spark_submit.py:514} INFO - at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:500)
[2021-12-29 12:02:54,888] {spark_submit.py:514} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
[2021-12-29 12:02:54,889] {spark_submit.py:514} INFO - at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[2021-12-29 12:02:54,890] {spark_submit.py:514} INFO - at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
[2021-12-29 12:02:54,891] {spark_submit.py:514} INFO - ... 1 more
[2021-12-29 12:03:14,328] {spark_submit.py:514} INFO - 21/12/29 12:03:14 ERROR StandaloneSchedulerBackend: Application has been killed. Reason: All masters are unresponsive! Giving up.
[2021-12-29 12:03:14,338] {spark_submit.py:514} INFO - 21/12/29 12:03:14 WARN StandaloneSchedulerBackend: Application ID is not initialized yet.
[2021-12-29 12:03:14,551] {spark_submit.py:514} INFO - 21/12/29 12:03:14 INFO SparkUI: Stopped Spark web UI at http://6bdda4c19a85:4053
[2021-12-29 12:03:14,660] {spark_submit.py:514} INFO - 21/12/29 12:03:14 INFO StandaloneSchedulerBackend: Shutting down all executors
[2021-12-29 12:03:14,725] {spark_submit.py:514} INFO - 21/12/29 12:03:14 INFO CoarseGrainedSchedulerBackend$DriverEndpoint: Asking each executor to shut down
[2021-12-29 12:03:14,922] {spark_submit.py:514} INFO - 21/12/29 12:03:14 WARN StandaloneAppClient$ClientEndpoint: Drop UnregisterApplication(null) because has not yet connected to master
[2021-12-29 12:03:14,977] {spark_submit.py:514} INFO - 21/12/29 12:03:14 INFO Utils: Successfully started service 'org.apache.spark.network.netty.NettyBlockTransferService' on port 44527.
[2021-12-29 12:03:14,983] {spark_submit.py:514} INFO - 21/12/29 12:03:14 INFO NettyBlockTransferService: Server created on 6bdda4c19a85:44527
[2021-12-29 12:03:15,086] {spark_submit.py:514} INFO - 21/12/29 12:03:15 INFO BlockManager: Using org.apache.spark.storage.RandomBlockReplicationPolicy for block replication policy
[2021-12-29 12:03:15,102] {spark_submit.py:514} INFO - 21/12/29 12:03:15 INFO BlockManagerMaster: Registering BlockManager BlockManagerId(driver, 6bdda4c19a85, 44527, None)
[2021-12-29 12:03:15,298] {spark_submit.py:514} INFO - 21/12/29 12:03:15 INFO MapOutputTrackerMasterEndpoint: MapOutputTrackerMasterEndpoint stopped!
[2021-12-29 12:03:15,496] {spark_submit.py:514} INFO - 21/12/29 12:03:15 INFO BlockManagerMasterEndpoint: Registering block manager 6bdda4c19a85:44527 with 434.4 MiB RAM, BlockManagerId(driver, 6bdda4c19a85, 44527, None)
[2021-12-29 12:03:15,527] {spark_submit.py:514} INFO - 21/12/29 12:03:15 INFO BlockManagerMaster: Registered BlockManager BlockManagerId(driver, 6bdda4c19a85, 44527, None)
[2021-12-29 12:03:15,540] {spark_submit.py:514} INFO - 21/12/29 12:03:15 INFO BlockManager: Initialized BlockManager: BlockManagerId(driver, 6bdda4c19a85, 44527, None)
[2021-12-29 12:03:15,708] {spark_submit.py:514} INFO - 21/12/29 12:03:15 INFO MemoryStore: MemoryStore cleared
[2021-12-29 12:03:15,730] {spark_submit.py:514} INFO - 21/12/29 12:03:15 INFO BlockManager: BlockManager stopped
[2021-12-29 12:03:15,896] {spark_submit.py:514} INFO - 21/12/29 12:03:15 INFO BlockManagerMaster: BlockManagerMaster stopped
[2021-12-29 12:03:15,899] {spark_submit.py:514} INFO - 21/12/29 12:03:15 INFO OutputCommitCoordinator$OutputCommitCoordinatorEndpoint: OutputCommitCoordinator stopped!
[2021-12-29 12:03:16,492] {spark_submit.py:514} INFO - 21/12/29 12:03:16 INFO SparkContext: Successfully stopped SparkContext
[2021-12-29 12:03:19,508] {spark_submit.py:514} INFO - 21/12/29 12:03:19 ERROR SparkContext: Error initializing SparkContext.
[2021-12-29 12:03:19,533] {spark_submit.py:514} INFO - java.lang.IllegalArgumentException: requirement failed: Can only call getServletHandlers on a running MetricsSystem
[2021-12-29 12:03:19,535] {spark_submit.py:514} INFO - at scala.Predef$.require(Predef.scala:281)
[2021-12-29 12:03:19,536] {spark_submit.py:514} INFO - at org.apache.spark.metrics.MetricsSystem.getServletHandlers(MetricsSystem.scala:89)
[2021-12-29 12:03:19,538] {spark_submit.py:514} INFO - at org.apache.spark.SparkContext.<init>(SparkContext.scala:603)
[2021-12-29 12:03:19,541] {spark_submit.py:514} INFO - at org.apache.spark.api.java.JavaSparkContext.<init>(JavaSparkContext.scala:58)
[2021-12-29 12:03:19,546] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2021-12-29 12:03:19,548] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[2021-12-29 12:03:19,557] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2021-12-29 12:03:19,559] {spark_submit.py:514} INFO - at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
[2021-12-29 12:03:19,564] {spark_submit.py:514} INFO - at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:247)
[2021-12-29 12:03:19,583] {spark_submit.py:514} INFO - at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:357)
[2021-12-29 12:03:19,595] {spark_submit.py:514} INFO - at py4j.Gateway.invoke(Gateway.java:238)
[2021-12-29 12:03:19,599] {spark_submit.py:514} INFO - at py4j.commands.ConstructorCommand.invokeConstructor(ConstructorCommand.java:80)
[2021-12-29 12:03:19,604] {spark_submit.py:514} INFO - at py4j.commands.ConstructorCommand.execute(ConstructorCommand.java:69)
[2021-12-29 12:03:19,612] {spark_submit.py:514} INFO - at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2021-12-29 12:03:19,637] {spark_submit.py:514} INFO - at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2021-12-29 12:03:19,648] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-29 12:03:19,690] {spark_submit.py:514} INFO - 21/12/29 12:03:19 INFO SparkContext: SparkContext already stopped.
[2021-12-29 12:03:19,700] {spark_submit.py:514} INFO - Traceback (most recent call last):
[2021-12-29 12:03:19,716] {spark_submit.py:514} INFO - File "/opt/***/jobs/get_ads_per_county.py", line 13, in <module>
[2021-12-29 12:03:19,724] {spark_submit.py:514} INFO - .appName("Get Ads per county") \
[2021-12-29 12:03:19,734] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/sql/session.py", line 228, in getOrCreate
[2021-12-29 12:03:19,757] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 392, in getOrCreate
[2021-12-29 12:03:19,777] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 147, in __init__
[2021-12-29 12:03:19,798] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 209, in _do_init
[2021-12-29 12:03:19,808] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py", line 329, in _initialize_context
[2021-12-29 12:03:19,813] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/py4j-0.10.9.2-src.zip/py4j/java_gateway.py", line 1574, in __call__
[2021-12-29 12:03:19,817] {spark_submit.py:514} INFO - File "/home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/py4j-0.10.9.2-src.zip/py4j/protocol.py", line 328, in get_return_value
[2021-12-29 12:03:19,826] {spark_submit.py:514} INFO - py4j.protocol.Py4JJavaError: An error occurred while calling None.org.apache.spark.api.java.JavaSparkContext.
[2021-12-29 12:03:19,839] {spark_submit.py:514} INFO - : java.lang.IllegalArgumentException: requirement failed: Can only call getServletHandlers on a running MetricsSystem
[2021-12-29 12:03:19,847] {spark_submit.py:514} INFO - at scala.Predef$.require(Predef.scala:281)
[2021-12-29 12:03:19,854] {spark_submit.py:514} INFO - at org.apache.spark.metrics.MetricsSystem.getServletHandlers(MetricsSystem.scala:89)
[2021-12-29 12:03:19,858] {spark_submit.py:514} INFO - at org.apache.spark.SparkContext.<init>(SparkContext.scala:603)
[2021-12-29 12:03:19,862] {spark_submit.py:514} INFO - at org.apache.spark.api.java.JavaSparkContext.<init>(JavaSparkContext.scala:58)
[2021-12-29 12:03:19,876] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2021-12-29 12:03:19,879] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[2021-12-29 12:03:19,883] {spark_submit.py:514} INFO - at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2021-12-29 12:03:19,890] {spark_submit.py:514} INFO - at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
[2021-12-29 12:03:19,904] {spark_submit.py:514} INFO - at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:247)
[2021-12-29 12:03:19,913] {spark_submit.py:514} INFO - at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:357)
[2021-12-29 12:03:19,919] {spark_submit.py:514} INFO - at py4j.Gateway.invoke(Gateway.java:238)
[2021-12-29 12:03:19,930] {spark_submit.py:514} INFO - at py4j.commands.ConstructorCommand.invokeConstructor(ConstructorCommand.java:80)
[2021-12-29 12:03:19,938] {spark_submit.py:514} INFO - at py4j.commands.ConstructorCommand.execute(ConstructorCommand.java:69)
[2021-12-29 12:03:19,945] {spark_submit.py:514} INFO - at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2021-12-29 12:03:19,951] {spark_submit.py:514} INFO - at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2021-12-29 12:03:19,958] {spark_submit.py:514} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2021-12-29 12:03:19,962] {spark_submit.py:514} INFO - 
[2021-12-29 12:03:21,789] {spark_submit.py:514} INFO - 21/12/29 12:03:21 INFO ShutdownHookManager: Shutdown hook called
[2021-12-29 12:03:21,790] {spark_submit.py:514} INFO - 21/12/29 12:03:21 INFO ShutdownHookManager: Deleting directory /tmp/spark-d1eea827-6bfa-402d-a982-a8797d3ce01f
[2021-12-29 12:03:21,791] {spark_submit.py:514} INFO - 21/12/29 12:03:21 INFO ShutdownHookManager: Deleting directory /tmp/spark-51b49c90-a62d-4d09-b120-81a938015d4d
[2021-12-29 12:03:25,077] {taskinstance.py:1703} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1332, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1458, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1514, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 179, in execute
    self._hook.submit(self._application)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 446, in submit
    f"Cannot execute: {self._mask_cmd(spark_submit_cmd)}. Error code is: {returncode}."
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://spark:7077 --conf spark.master=spark://spark:7077 --name scrap data --verbose /opt/***/jobs/get_ads_per_county.py mayo. Error code is: 1.
[2021-12-29 12:03:25,355] {taskinstance.py:1280} INFO - Marking task as UP_FOR_RETRY. dag_id=scraping_daft_mayo, task_id=scrap_data, execution_date=20211228T102000, start_date=20211229T115735, end_date=20211229T120325
[2021-12-29 12:03:25,552] {standard_task_runner.py:91} ERROR - Failed to execute job 5333 for task scrap_data
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/task/task_runner/standard_task_runner.py", line 85, in _start_by_fork
    args.func(args, dag=self.dag)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/cli_parser.py", line 48, in command
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/utils/cli.py", line 92, in wrapper
    return f(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 292, in task_run
    _run_task_by_selected_method(args, dag, ti)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 107, in _run_task_by_selected_method
    _run_raw_task(args, ti)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 184, in _run_raw_task
    error_file=args.error_file,
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/utils/session.py", line 70, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1332, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1458, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1514, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 179, in execute
    self._hook.submit(self._application)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 446, in submit
    f"Cannot execute: {self._mask_cmd(spark_submit_cmd)}. Error code is: {returncode}."
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://spark:7077 --conf spark.master=spark://spark:7077 --name scrap data --verbose /opt/***/jobs/get_ads_per_county.py mayo. Error code is: 1.
[2021-12-29 12:03:25,727] {local_task_job.py:154} INFO - Task exited with return code 1
[2021-12-29 12:03:27,209] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
